test/docs.js

```diff
@@ -30,6 +35,37 @@ describe('Docs', function() {
               }
             }
           }
+        },
+        '@apostrophecms/migration': {
+          extendMethods(self) {
+            return {
+              add(_super, name, migrationFn, options) {
+                migrations.addList.push(name);
+                migrationFn();
+                return _super(name, migrationFn, options);
+              },
+              eachDoc(_super, criteria, limit, iterator) {
+                if (criteria.cacheInvalidatedAt) {
```

Comment:
This is going to alter the behavior of every eachDoc call in every migration, which could be quite a lot of unexpected behavior.

Also your `add` call adds a migration for every other migration.

You should call `self.apos.migration.add` in one of the modules of your test, i.e. call that from `init` of that module or some method invoked by `init`. 

Here is an example:

https://github.com/apostrophecms/apostrophe/blob/main/modules/%40apostrophecms/area/index.js#L581-L606

Reply:
Your migration will call the regular `eachDoc` method. There is no need to alter the behavior of all `eachDoc` methods for every migration, unless I am missing something.

Reply:
I see that you wrote a new migration which is correct, but this code in the tests to override all migrations in a strange way is still here. I think you meant to remove it?